testsuite/features/support/constants.rb

```diff
@@ -1011,15 +1013,13 @@
         ubuntu-2404-amd64-main-security-amd64
         ubuntu-24.04-suse-manager-tools-amd64
       ],
-    'suma-proxy-extension-50' => # CHECKED
+    'suse-multi-linux-manager-proxy-5.1' => # CHECKED
```

Comment:
5.1 -> 51 (?)

normally we try to have same constants across branches, so maybe you could add instead of replacing